Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -0,0 +1,51 @@
+public class Game {
+	var word: String
+	let guessManager: GuessManager
+	var view: View
+
+	public init(word: String, guessManager: GuessManager, view: View) {
+		self.word = word
+		self.guessManager = guessManager
+		self.view = view
+	}
+
+	public func playGame() {
+		while (!isGameOver()) {
+			playerTurn()
+		}
+		displayResults()
+	}
+
+	public func isGameOver() -> Bool {
+		return guessManager.hasNoGuessesRemaining() || isWinner()
+	}
+
+	public func isWinner() -> Bool {
+		let letters = separateLetters()
+		return guessManager.determineUnguessedLetters(letters).isEmpty || guessManager.correctlyGuessedFullWord(word)
+	}
+
+	private func playerTurn() {		
+		view.assignBlanks(word, correctGuesses: guessManager.correctGuesses)
+		guessManager.guess = guessManager.assignGuess(view)
```

Comment:
What I meant as assignGuess was assigning the guess to the instance variable `guess` of the `guessManager` class. We might have to talk this one out. See my note regarding holding `guess` state in `guessManager`